src/arch/Sound/DSoundHelpers.cpp

```diff
@@ -42,7 +42,7 @@ BOOL CALLBACK DSound::EnumCallback( LPGUID lpGuid, LPCSTR lpcstrDescription, LPC
 void DSound::SetPrimaryBufferMode()
 {
 	DSBUFFERDESC format;
-	memset( &format, 0, sizeof(format) );
+	ZeroMemory( &format, sizeof(format) );
```

Comment:
Isn't ZeroMemory a windows only macro? Is this file windows only (I can't tell).

Reply:
It is, yes, but this is Windows-only (it's DirectSound) so it's safe to use here.